rpcpb/rpc.proto

```diff
@@ -152,15 +152,18 @@ message ClusterInfo {
   // Maps from the node ID to its attached peer infos.
   map<string, ListOfAttachedPeerInfo> attached_peer_infos = 6;
 
-  // Set to "true" once custom VMs are ready.
-  bool custom_vms_healthy = 7;
-  // The map of custom VM IDs in "ids.ID" format to its VM information.
-  map<string, CustomVmInfo> custom_vms = 8;
+  // Set to "true" once custom blockchains are ready.
+  bool custom_chains_healthy = 7;
+  // The map of blockchain IDs in "ids.ID" format to its information.
```

Comment:
Nit: this sentence sounds strange now, "to its information" refers to what?

Reply:
went back to previous sentence, replacing vm with blockchain